wicketd/src/update_tracker.rs

```diff
@@ -946,6 +977,78 @@ impl UpdateDriver {
                 },
             )
             .register();
+
+        // Send the bootloader update to the RoT.
+        // XXX is this where we mutex?
```

Comment:
Wicket is all manually driven, right? By that I mean that each stage0 update needs to be triggered by a human.
In any case, we don't want more than one stage0 update in process at any one time and we want manual intervention if any fail. If we loose a device because of a bricked RoT, we don't want to proceed.
This isn't a real issue until the chain of trust and trust quorum are fully built out, but bricked RoTs are going to at least require field service to touch the machines.

Reply:
> Wicket is all manually driven, right? By that I mean that each stage0 update needs to be triggered by a human.

Wicket is only manually driven to a point. A human kicks off "do all the updates for sled N", but then wicket drives RoT -> SP -> host updates autonomously. Multiple sleds can be updated in parallel.